### DISCUSSION TITLE
Traceback (most recent call last):

### DISCUSSION THREAD

**Discussion by Therealfredd** [abey79/vsketch#371] [Q&A]
After doing basic installation and trying to init or run I get the same error. I have removed and reinstalled multiple times and in differnt directories as well as differnt terminals (pycharn, vscode, wsl) and no luck. ##

**the command run was:**

`vsk run examples/schotter`

**Error:**

> Traceback (most recent call last):
>   File "/mnt/c/Users/USER/Desktop/grid_drawings/vsketch/venv/bin/vsk", line 5, in <module>
>     from vsketch_cli.cli import cli
>   File "/mnt/c/Users/USER/Desktop/grid_drawings/vsketch/venv/lib/python3.10/site-packages/vsketch_cli/cli.py", line 15, in <module>
>     from .gui import show
>   File "/mnt/c/Users/USER/Desktop/grid_drawings/vsketch/venv/lib/python3.10/site-packages/vsketch_cli/gui.py", line 6, in <module>
>     import qasync
>   File "/mnt/c/Users/USER/Desktop/grid_drawings/vsketch/venv/lib/python3.10/site-packages/qasync/__init__.py", line 85, in <module>
>     QtGui = importlib.import_module(QtModuleName + ".QtGui", package=QtModuleName)
>   File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
>     return _bootstrap._gcd_import(name[level:], package, level)

**Comment by abey79** [comment]
Which OS are you using? How did you install the Python interpreter (`/usr/lib/python3.10/*`) you are using? How did you create the venv in which vsketch was installed? Did you try to install with pipx, as recommended by the [doc](https://vsketch.readthedocs.io/en/latest/install.html#installing-using-pipx)?
